src/components/molecules/TodoActions.vue

```diff
@@ -7,39 +7,74 @@ const store = useStore()
 
 const filters = computed(() => store.getters.getFilters)
 const remaining = computed(() => store.getters.undoneTodosCount)
+const visibility = computed(() => store.getters.getVisibility)
 const clearCompleted = () => store.dispatch('clearCompleted')
 const changeVisibility = (visibility) => store.dispatch('changeVisibility', visibility)
 
 const capitalize = (s) => s.charAt(0).toUpperCase() + s.slice(1)
+
+const remainingText = (remaining) => {
```

Comment:
It looks like you don't need an argument in this function, you can use remaining computed itself. Please, make it a computed, not a method.